relay-server/src/services/processor.rs

```diff
@@ -1591,7 +1592,12 @@ impl EnvelopeProcessorService {
 
         if state.has_event() {
             event::scrub(state, project_info.clone())?;
-            event::serialize(state, event_fully_normalized)?;
+            event::serialize(
+                state,
+                event_fully_normalized,
+                EventMetricsExtracted(false),
+                SpansExtracted(false),
```

Comment:
Assuming errors do not have extraction of metrics and spans, I purposefully put them here to make it more clear that these are always `false`.

Reply:
It would be nice in the future to make the functions a bit more independent for each event type, but it's a limitation of our event model.